flow-typed/types/ima-config.js

```diff
@@ -37,6 +46,7 @@ type _ImaConfigObject = {
   disableMediaPreload: boolean,
   setDisableCustomPlaybackForIOS10Plus: boolean,
   adsRenderingSettings: Object,
+  adsRequest: Object,
```

Comment:
I think it's not good to expose the `adsRequest` because in ima model it contains the `adTagUrl` but in our model `adTagUrl` is in the root config. it will be confused where to config `adTagUrl`.
maybe expose `vastLoadTimeout` only in the root config. @OrenMe What do you think?

Reply:
Agree